woocommerce/payment-gateway/assets/js/frontend/sv-wc-payment-gateway-payment-form.coffee

```diff
@@ -382,3 +382,7 @@ jQuery( document ).ready ($) ->
 		#
 		# @since 3.0.0
 		unblock_ui: -> @form.unblock()
+
+
+	# dispatch loaded event
+	$( document.body ).trigger( "wc_#{ @plugin_id }_payment_form_handler_loaded" )
```

Comment:
@wvega, I got the event name from the page source, but I thought it was a bit weird that this event was different from the others. Is it correct?

Reply:
The `wc_authorize_net_payment_form_handler_loaded` event name in this case is a side effect of the overwritten `get_js_handler_class_name()` method in `woocommerce-gateway-authorize-net-cim/includes/Payment_Form.php`. 

I believe it's correct that `render_js()` is using that event name because Authorize.Net initializes a subclass of the handler included in the framework (`WC_Authorize_Net_Payment_Form_Handler`) that needs to be loaded separately.

However, `sv-wc-payment-gateway-payment-form.coffee` should probably still trigger `sv_wc_payment_form_X_Y_Z_loaded` so that it works on gateways that use the base handler directly.

In order to upgrade Authorize.Net to use this version of the framework, we would also need to trigger `wc_authorize_net_payment_form_handler_loaded` in `woocommerce-gateway-authorize-net-cim/assets/js/frontend/wc-authorize-net-cim.coffee`.

Reply:
That makes sense, I've fixed the event for this branch.